Composer/packages/client/src/pages/setting/dialog-settings/DialogSettings.tsx

```diff
@@ -37,9 +36,6 @@ export const DialogSettings: React.FC<RouteComponentProps> = () => {
   const { setSettings, setLocale, addLanguageDialogBegin } = useRecoilValue(dispatcherState);
 
   const { languages, defaultLanguage } = settings;
-  useEffect(() => {
-    if (!botName) navigateTo('/home');
```

Comment:
No bot opened , navigate to Home. No needed anymore cause we also fetchProject in setting page.